### ISSUE TITLE
Scaling model missing some expressions

### ISSUE THREAD

**Issue by jpfairbanks** [jpfairbanks/SemanticModels.jl#101]
@infvie @kuncao0228 I was working with the SEIR model example notices something missing: 

```julia
function micro_1(du, u, parms, time)
    # PARAMETER DEFS
    # β transmition rate
    # r net population growth rate
    # μ hosts' natural mortality rate
    # Κ population size
    # α disease induced mortality rate

    β, r, μ, K, α = parms
    dS = r*(1-S/K)*S - β*S*I
    dI = β*S*I-(μ+α)*I
    du = [dS,dI]
end
```
Is there something I am missing here?

**Comment by infvie** [comment]
I'll check against the cookbook and fix it

**Comment by infvie** [comment]
It looks like this is the same. http://epirecip.es/epicookbook/chapters/deleo1996/julia

**Comment by jpfairbanks** [comment]
It was inside a macro definition in the cookbook, but now is using the macro-free api. fixed in #100

**Comment by Quar** [comment]
Sorry for the trouble, but just in case it worths noting for beginners -- it seems the Jupyter Notebook at
[`SemanticModels.jl/examples/epicookbook/notebooks/SimpleDeterministicModels/ScalingModel.ipynb`](https://github.com/jpfairbanks/SemanticModels.jl/blob/8d4a4b3b079e1c13baee8dc9db4b3d460197b4bd/examples/epicookbook/notebooks/SimpleDeterministicModels/ScalingModel.ipynb)

is not updated with [`ScalingModel.jl`](https://github.com/jpfairbanks/SemanticModels.jl/blob/8d4a4b3b079e1c13baee8dc9db4b3d460197b4bd/examples/epicookbook/src/ScalingModel.jl) (still missing [`S, I = u`](https://github.com/jpfairbanks/SemanticModels.jl/blob/8d4a4b3b079e1c13baee8dc9db4b3d460197b4bd/examples/epicookbook/src/ScalingModel.jl#L28)).

Thanks enormously for the abundance of great examples and demos by the way!
